### PR TITLE
feat: Source connector for RMS Cloud

### PR DESCRIPTION
**Pull request by cjrh**

### CODE REVIEW

File: airbyte-integrations/connectors/source-kapiche-rms-cloud-api/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Kapiche Export API Source
```
Comment (wallies) [inline airbyte-integrations/connectors/source-kapiche-rms-cloud-api/README.md:1]:
```suggestion
# RMS API Source
```

File: airbyte-integrations/connectors/source-kapiche-rms-cloud-api/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+This is the repository for the Kapiche Export Api source connector, written in Python.
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/sources/kapiche-export-api).
+
+## Local development
+
+### Prerequisites
+**To iterate on this connector, make sure to complete this prerequisites section.**
+
+#### Minimum Python version required `= 3.9.0`
```
Comment (wallies) [inline airbyte-integrations/connectors/source-kapiche-rms-cloud-api/README.md:11]:
```suggestion
#### Minimum Python version required `= 3.10.0`
```

File: airbyte-integrations/connectors/source-kapiche-rms-cloud-api/integration_tests/catalog.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "TODO fix this file",
```
Comment (wallies) [inline airbyte-integrations/connectors/source-kapiche-rms-cloud-api/integration_tests/catalog.json:4]: ??

File: airbyte-integrations/connectors/source-kapiche-rms-cloud-api/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+RUN echo "Etc/UTC" > /etc/timezone
+
+# copy payload code only
+COPY main.py ./
+COPY source ./source
+
+ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
+ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
+
+LABEL io.airbyte.version=0.1.0export-api
```
Comment (wallies) [inline airbyte-integrations/connectors/source-kapiche-rms-cloud-api/Dockerfile:33]:
```suggestion
LABEL io.airbyte.version=0.2.0
```

File: airbyte-integrations/connectors/source-kapiche-rms-cloud-api/integration_tests/catalog.json (hunk 3 above)
Comment (cjrh) [reply airbyte-integrations/connectors/source-kapiche-rms-cloud-api/integration_tests/catalog.json:4]: I haven't used this file for anything, it copied over like this from the export api project. Should I do anything with this?

Comment (wallies) [reply airbyte-integrations/connectors/source-kapiche-rms-cloud-api/integration_tests/catalog.json:4]: Ok I guess just put a note not used

**Review by wallies — APPROVED** [review]

**Pull request merged**